linker/Linker.Steps/MarkStep.cs

```diff
@@ -131,7 +133,7 @@ void Process ()
 			if (QueueIsEmpty ())
 				throw new InvalidOperationException ("No entry methods");
 
-			while (ProcessPrimaryQueue () || ProcessLazyAttributes ())
+			while (ProcessPrimaryQueue () || ProcessLazyAttributes () || ProcessLateMarkedAttributes ())
```

Comment:
What's lazy attributes? It sounds similar.

Reply:
It is very similar.  Those are Assembly and Module attributes.  Awhile back I put in a special rule for these attributes because there were cases where we would keep a reference to an assembly all because of an attribute defined in that assembly.

I didn't see an easy way to unify the two.  And for the moment, when this new option is not enabled, I do not want there to be a behavior change.

That's for pointing this out, I realized these assembly and module attribute's were not following the behavior of this new option.